### ISSUE TITLE
[Feature request] Ability to pass-protect only some slots

### ISSUE THREAD

**Issue by kirillmukhin** [TF2Stadium/QA#43]
If you have few (less than 12 for 6s, 18 for HL) friends ready to play the lobby, right now you have two ways to create it:
1. Put password on all slots and copy it to the chat, so other people can join. Which is confusing for audience and lobby probably will not be filled.
2. Make lobby without a password; and kick everyone who has taken slot(-s) desired by friend(-s). Which isn't polite in my opinion.

Ability to reserve only several slots at the start with option to un-reserve them later would be great.

**Comment by gpittarelli** [comment]
This is done (a few weeks ago), albeit in a slightly roundabout way: you have to make the lobby (either initially all password protected, or initially none password-protected) and then once its made you can, using the menus on each slot, remove/change/add passwords and other restrictions per-slot.

A better lobby creation wizard so you can make these per-slot changes before starting the lobby is planned, but low priority for now.

As always, contributions (such as UI design mockups) are greatly appreciated...
